src/test/unzip.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { expect } from "chai";
+import * as fs from "fs";
+import * as path from "path";
+import { unzip } from "../unzip";
+import { DownloadDetails } from "../emulator/downloadableEmulators";
+import { Client } from "../apiv2";
+const fixturesDir = path.resolve(__dirname, "./fixtures");
+
+const ZIP_FIXTURES_PATH = path.join(fixturesDir, "zip-files");
+const ZIP_TEMPORARY_PATH = path.join(ZIP_FIXTURES_PATH, "temp");
+const ZIP_UNZIPPER_PATH = path.join(ZIP_FIXTURES_PATH, "node-unzipper-testData");
+
+describe("unzip", () => {
+  let uiZipPath: string;
+  let pubsubZipPath: string;
+  before(function (done) {
+    // eslint-disable-next-line @typescript-eslint/no-invalid-this
+    this.timeout(5000);
+    (async () => {
```

Comment:
Unclear to me why we need this needs to be set up as a self executing function. Could this just be:
```
before(async () => {
 ...
}

```